assam_budget_board/variables.py

```diff
@@ -0,0 +1,15 @@
+be  =  "Budget 2019-20"
```

Comment:
Can we move these to settings.py? Also, can we use the variables formats as in settings.py as the standard nomenclature. Also, "_" and CamelCase don't go together.

Reply:
@shreyaagrawal0809 Do we still need this file? Otherwise git rm this file

Reply:
Done